src/modules/household/components/elements/ConflictingEnrollmentAlert.tsx

```diff
@@ -0,0 +1,106 @@
+import {
```

Comment:
should this go inside the "householdActions" dir which contains the merge/split-related components?

Reply:
Ok, sounds good -- I also had a broader question about the directory structure here: https://github.com/greenriver/hmis-frontend/pull/1022#discussion_r1920527491

I remember at one point the floating idea that we want to keep module structure relatively flat, I think it was for the sake of Storybook?

Reply:
Yeah, we could probably flatten those out. It looks like several components in that elements directory are not used at all.